server/server.go

```diff
@@ -53,12 +56,14 @@ type Server struct {
 	engine      *engine.Engine
 	mach        *machine.CoreOSMachine
 	hrt         heart.Heart
-	mon         *heart.Monitor
+	mon         *Monitor
 	api         *api.Server
 
 	engineReconcileInterval time.Duration
 
-	stop chan bool
+	killc chan bool      // used to signal monitor to shutdown server
```

Comment:
I'd love better names here since this isn't actually kill vs. stop. Maybe `triggerStopc` and `shutdownc`?